cognite/client/data_classes/data_modeling/containers.py

```diff
@@ -276,8 +276,8 @@ def dump(self, camel_case: bool = True) -> dict[str, str | dict]:
         output["immutable"] = self.immutable
         for key in ["nullable", "auto_increment", "name", "default_value", "description"]:
             if (value := getattr(self, key)) is not None:
-                output[key] = value
-        return convert_all_keys_to_camel_case_recursive(output) if camel_case else output
```

Comment:
This is a dangerous function. Should we have a task to remove it? Or do it have any good use cases?

Reply:
Agreed, I had the same thought!